src/pyDataverse/docs/source/user/basic-usage.rst

```diff
@@ -72,10 +72,20 @@ can then be used (e. g. :meth:`json() <requests.Response.json>`).
 Create Dataverse Collection
 -----------------------------
 
-The top-level data-type in the Dataverse software is called a Dataverse collection, so we will
-start with that.
+The top-level data-type in the Dataverse software is called a Dataverse collection, so we will start with that. 
+Take a look at the figure below to better understand the relationship between a Dataverse collection, a dataset, and a datafile. 
 
-First, instantiate a :class:`Dataverse <pyDataverse.models.Dataverse>`
+.. figure:: ../_images/collection_dataset.png
```

Comment:
It is a great idea, to show the relations visually. Some thoughts: it's not obvious from the diagram, that a dataverse could also have several dataverse inside, or more than one dataset (as it is clear for the datafiles).